libcodechecker/libhandlers/analyze.py

```diff
@@ -176,6 +176,18 @@ def add_arguments_to_parser(parser):
                              "reports and overwrites only those files that "
                              "were update by the current build command).")
 
+    parser.add_argument('--context-free-hash',
```

Comment:
Maybe it would be better to support hash versioning on the long term in the command line.
We could use choices as values and v1 would be the default.
```choices=('v1')```

or rename the argument to `--hash`/`--report-hash` and for the choices
```choices=('context-free-v1')``` or ```choices=('context-free')```